.prettierrc.json

```diff
@@ -1,9 +0,0 @@
-{
-    "tabWidth": 2,
-    "useTabs": false,
-    "semi": true,
-    "singleQuote": true,
-    "printWidth": 80,
-    "plugins": ["prettier-plugin-svelte"]
-  }
```

Comment:
we also want to keep these styling rules for React. But the `plugins` can be removed.